CHANGELOG-2.2.md

```diff
@@ -0,0 +1,13 @@
+# Changelog since v2.1.0
+
+### Bug Fixes
+
+- A bug that prevented the external-attacher from releasing its finalizer from a `PersistentVolume` object that was created using a legacy storage class provisioner has been fixed. ([#218](https://github.com/kubernetes-csi/external-attacher/pull/218), [@rfranzke](https://github.com/rfranzke))
```

Comment:
prefix that it is related to the csi migration feature to make it more obvious

Reply:
Added "... using a legacy storage class provisioner and *migrated to CSI*"